### PR TITLE
adjusted script to use cothority-test-scripts

### PR DESCRIPTION
**Pull request by ineiti**
Finally not a bad error - just needed a proper error-handling in the api. Also updated the script to use the cothority-test-scripts.

### CODE REVIEW

File: test_cosi.sh [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
-# Set to a non-empty value to print debugging messages
-DBG_RUN=1
-. $GOPATH/src/gopkg.in/dedis/cothority.v0/app/libtest.sh
+DBG_SHOW=1
+# Debug-level for app
+DBG_APP=2
+# Uncomment to build in local dir
 STATICDIR=test
 
+. $GOPATH/src/github.com/dedis/cothority/app/lib/test/libtest.sh
```
Comment (nikkolasg) [inline test_cosi.sh:9]: Are we sure the file exists  when running the script ? `dedis/cosi` use `app/client/lib/config` but that's it. Just asking, I did not check in depth...

Comment (ineiti) [inline test_cosi.sh:9]: yes, it should exist. `cosi` depends on `cothority`, so that file should be there.

**Comment by nikkolasg** [comment]
Waiting on fixes for Travis, otherwise OK to merge.

**Comment by ineiti** [comment]
OK, linting is good.

**Pull request merged**